homework-autocomplete-ui/dist/bundle.js

```diff
@@ -0,0 +1 @@
.bind(null,e,t),i=function(){!function(n){if(null===n.parentNode)return!1;n.parentNode.removeChild(n)}(e)};return r(n),function(t){if(t){if(t.css===n.css&&t.media===n.media&&t.sourceMap===n.sourceMap)return;r(n=t)}else i()}}n.exports=function(n,t){(t=t||{}).attributes="object"==typeof t.attributes?t.attributes:{},t.singleton||"boolean"==typeof t.singleton||(t.singleton=o());var e=s(n,t);return u(e,t),function(n){for(var r=[],o=0;o<e.length;o++){var a=e[o],c=i[a.id];c&&(c.refs--,r.push(c))}n&&u(s(n,t),t);for(var l=0;l<r.length;l++){var d=r[l];if(0===d.refs){for(var f=0;f<d.parts.length;f++)d.parts[f]();delete i[d.id]}}}}},function(n,t,e){"use strict";e.r(t);var r=function(n,t){"string"==typeof n&&"string"==typeof t||console.log("tag and tagClass should be a string");const e=document.createElement(n);return e.classList.add(t),e};var i=function(){const n=r("main","wrapper"),t=r("div","container"),e=r("div","inputContainer"),i=r("input","inputContainer__input"),o=r("label","inputContainer__label"),a=document.createTextNode("SEARCH");i.setAttribute("id","input"),i.setAttribute("type","text"),i.setAttribute("name","search"),o.setAttribute("for","input"),o.appendChild(a),e.appendChild(o),e.appendChild(i),t.appendChild(e),n.appendChild(t),document.body.appendChild(n)};var o=function(){const n=document.querySelector(".container"),t=r("div","resultContainer"),e=r("div","resultContainer__title"),i=r("div","resultContainer__text"),o=document.createTextNode("RESULT");e.appendChild(o),i.setAttribute("id","result"),t.appendChild(e),t.appendChild(i),n.appendChild(t)};var a=class{constructor(n){this.key=n,this.parent=null,this.children={},this.end=!1,this.count=null}};var s=function(n){const t=[];let e=n;for(;null!==e;)t.push(e.key),e=e.parent;return t.reverse(),t.join("")};var u=function n(t,e=[]){const r=e;if(t.end)for(let n=0;n<t.count;n++)r.push(s(t));for(const e in t.children)n(t.children[e],r);return r};var c=class{constructor(){this.root=new a(null)}insert(n,t){let e=this.root;for(let r=0;r<n.length;r++)e.children[n[r]]||(e.children[n[r]]=new a(n[r]),e.children[n[r]].parent=e),e=e.children[n[r]],r==n.length-1&&(e.end=!0,e.count=t)}find(n){let t=this.root;const e=[];for(let r=0;r<n.length;r++){if(!t.children[n[r]])return e;t=t.children[n[r]]}return u(t)}};var l=function(n,t=!0){return t?n.charAt(0).toUpperCase()+n.slice(1):n.charAt(0).toLowerCase()+n.slice(1)};var d=function(n){const t=new c,e=n.reduce((n,t)=>(n[t]||(n[t]=0),n[t]++,n),{});return[...new Set(n)].forEach(n=>{const r=e[n];t.insert(n,r)}),function(n){if(!n||""===n)return[];const e=l(n,!1),r=l(n),i=t.find(e),o=t.find(r);return i.concat(o)}};var f=function(n,t,e){const r=document.querySelector(n),i=document.querySelector(t),o=d(e);r.addEventListener("input",()=>{i.innerHTML=o(r.value)})};var p=["Anton","Andrey","Boris","Vlad","Denis","Diana","Alina","Avgust","Band","Minsk","Mogilev","Grodno","Gomel","Vitebsk","Pinsk","Polock","Vilnus"];var h=function(){i(),o(),f("#input","#result",p)};e(0);h()}]);
```

Comment:
As a rule bundle should be in .gitignore. If anybody need in it, he should make build by himself.